index.html

```diff
@@ -260,7 +260,7 @@ <h3><code>#initial</code></h3>
       <h3><code>#textShadow</code></h3>
 
       <p>To support the edge styles required by the FCC, [[!ttml-imsc1.1]] Text Profile SHALL support the <code>#textShadow</code>
-      feature specified in [[!TTML2]].</p>
+      feature specified in [[!TTML2]], excluding support for blur radius values other than <code>0</code>.</p>
```

Comment:
I'd prefer there to be a `#textShadow-blurred` feature in TTML2, but in the meantime this would do.